app/views/shared/_status.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="<%= dialog_column_class %>">
```

Comment:
Does this really belong in `shared`, or in `requests` (where Rails templating could automatically discover it..)?